src/core.js

```diff
@@ -166,11 +242,11 @@ async function pruneNonCriticalCssLauncher ({
       )
       debuglog('build selector profile')
 
+      await page.waitFor(500)
```

Comment:
this is the kind of change that I most like would want as an opt-in, or even just configurable so that only users who need to change this can do so, without slowing down penthouse for everyone.

Reply:
Maybe you are right. I need to use it in a very large scale and Iam in need of a working system. If there is a chance of an error my results will be wrong and this is crucial and could cost thousands of Euros.
But of course, we could make an options out of it. Good idea